README.md

```diff
@@ -38,6 +38,20 @@ For each project:
    6. Ensure the "Enable Code Analysis" and "Suppress results from generated code" options are enabled on your project.
 2. Add the `StyleCop.Analyzers` NuGet package to your project. This will enable style warnings.
 
+### EditorConfig
```

Comment:
Rusty, can you also clean up the instructions on lines 18-20 above?

Reply:
Cleaned up in #15.